src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+/* Styles */
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", "Montserrat", sans-serif;
+  font-size: 12px;
+}
+
+/* main header decoration: */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  background-color: white;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.logo-link {
+  /*because of 3 strange pixels below the logo link*/
+  margin-bottom: -3px;
+}
```

Comment:
Looks like `kostyl` let's set flex to this item, it will help
Also, what about default text-decoration?